client/src/res/values/joyride.js

```diff
@@ -50,18 +50,30 @@ export const JOYRIDE_STEPS = [
           target: '.leaflet-states-pane .leaflet-interactive:nth-child(4)',
           content: 'Click on California to unlock it and then click "Next"',
           spotlightClicks: true,
+          disableBeacon:true,
+        },// At the end of step 7, need to jump zoom to la county
+        { //8
+          target: '.leaflet-counties-pane .leaflet-interactive:nth-child(240)',
+          content: 'Incidents can also be viewed at a county level with the same functionality as states. Zoom in to see data for individual counties.',
+          spotlightClicks: true,
+          disableBeacon: true,
+        },
+        {// 9
+          target: '.MuiToggleButtonGroup-root',
+          content: 'Use this location switcher to jump to view incidents within Alaska and Hawaii',
+          spotlightClicks: true,
         },
-        {// 8
+        {// 10
           disableBeacon: false,
           target: '#covidButton',
           content: 'View and report Asian American/Pacific Islander hate crimes from COVID-19 discrimination here.',
         },      
```

Comment:
When I try to click on the link, it ends (skips) the tutorial. Possible UX improvement by either disabling or connecting the tutorial onto the COVID page ("Click to go back")?

Reply:
Hm I see. Because there's an overlay enabled on this step, clicking anywhere will cause the tutorial to close

Reply:
I like the idea of connecting the tutorial to the Covid page. Maybe could add a prop to the covid page that if the tutorial was currently running, assume (or force) that the user clicked the covid page during the step highlighting it. Make a popup that gives the option to go back and continue the tutorial or otherwise end it

Reply:
Yeah, you could also look into contexts, that may or may not be cleaner (haven't really used them). I think it's similar to Redux, without needing to install a whole new dependency